_posts/2013-05-07-icra-motion-planning-tutorial.markdown

```diff
@@ -56,7 +56,7 @@ You can register for the tutorial when you register through the ICRA site. Howev
 **Schedule**
 
 * 08:45 - 09:00 - Welcome message, Overview of tutorial - [PDF]({{ site.url }}/assets/pdfs/2013/icra2013tutorial/ICRATutorialOverview.pdf)
-* 09:00 - 09:20 - MoveIt! - (Sachin Chitta) [PDF]({{ site.url }}/assets/pdfs/2013/icra2013tutorial/ICRATutorial-MoveIt.pdf)
+* 09:00 - 09:20 - MoveIt - (Sachin Chitta) [PDF]({{ site.url }}/assets/pdfs/2013/icra2013tutorial/ICRATutorial-MoveIt.pdf)
```

Comment:
I think here the historical name should be kept, maybe even in all blog posts?

Reply:
I've restored all blog posts to keep MoveIt!